python/python/glide/async_commands/cluster_commands.py

```diff
@@ -478,3 +483,34 @@ async def publish(self, message: str, channel: str, sharded: bool = False) -> in
             RequestType.SPublish if sharded else RequestType.Publish, [channel, message]
         )
         return cast(int, result)
+
+    async def flushall(
+        self, flush_mode: Optional[FlushMode] = None, route: Optional[Route] = None
+    ) -> TClusterResponse[TOK]:
+        """
+        Deletes all the keys of all the existing databases. This command never fails.
+
+        See https://valkey.io/commands/flushall for more details.
+
+        Args:
+            flush_mode (Optional[FlushMode]): The flushing mode, could be either `SYNC` or `ASYNC`.
+            route (Optional[Route]): The command will be routed to a random node, unless `route` is provided,
+                in which case the client will route the command to the nodes defined by `route`.
+
+        Returns:
+            str: OK.
```

Comment:
Copy from another command if client has one with the same return type
```suggestion
            TClusterResponse[TOK]: OK.
```